src/Api/API.php

```diff
@@ -1272,7 +1272,7 @@ protected function getItems($itemtype, $params = [], &$totalcount = 0)
         }
 
        // get result full row counts
-        $count_query = "SELECT COUNT(*) FROM {$DB->quoteName($table)} $join WHERE $where";
+        $count_query = "SELECT " . QueryFunction::count('*') . " FROM {$DB::quoteName($table)} $join WHERE $where";
         $totalcount = $DB->query($count_query)->fetch_row()[0];
```

Comment:
IMHO, in this kind of cases, we should probably replace usage of `$DB->query()` by `$DB->request()`.